docs/content/configuration/csi_ug_config_create_secret_topology.md

```diff
@@ -0,0 +1,48 @@
+# Creating a Secret with topology awareness
+
+Create an array secret YAML file in order to define the storage credentials (username and password) and address. Use this information for creating a Secret that is topology aware.
+
+**Note:** If you are not using the CSI Topology feature, follow the steps in [Creating a Secret](csi_ug_config_create_secret.md).
+
+Use the Secret file to connect a worker node to a storage system.
+
+Within the Secret, each `management-id` (here, represented by `demo-management-id-x`), is used to identify the storage system within other configuration files.
+
+1. Create the secret file, similar to the following demo-secret-config.json:
+
+    The `management_address` field can contain more than one address, with each value separated by a comma.
+
+    ```
+      {
+    "demo-management-id-1": {
+      "username": "demo-username-1",
+      "password": "demo-password-1",
+      "management_address": "demo-management-address-1",
+      "supported_topologies": [
+        {
+          "topology.block.csi.ibm.com/demo-region": "demo-region-1",
+          "topology.block.csi.ibm.com/demo-zone": "demo-zone-1"
+        }
+      ]
+    },
+    "demo-management-id-2": {
+      "username": "demo-username-2",
+      "password": "demo-password-2",
+      "management_address": "demo-management-address-2",
+      "supported_topologies": [
+        {
+          "topology.block.csi.ibm.com/demo-region": "demo-region-2",
+          "topology.block.csi.ibm.com/demo-zone": "demo-zone-2"
+        }
+      ]
+    }
+  }
+     ```
```

Comment:
this appears to be outside the block formatting

Reply:
DONE